doc/userguide/configuration/suricata-yaml.rst

```diff
@@ -268,7 +268,7 @@ integration with 3rd party tools like logstash.
       #  server: 127.0.0.1
       #  port: 6379
       #  async: true ## if redis replies are read asynchronously
-      #  mode: list ## possible values: list (default), channel
+      #  mode: list ## possible values: list (default), rlist (uses RPUSH instead of LPUSH), channel
```

Comment:
I agree that "list" was not the perfect name in the first place. Like the idea to use "lpush" and "rpush" while keeping "list" as an "alias" for "lpush" for backwards compatibility.